benchexec/tablegenerator/test_util.py

```diff
@@ -98,3 +98,31 @@ def test_print_decimal_float(self):
             expected = str(float(value))
             assert "e" not in expected, expected
             self.assertEqual(expected, util.print_decimal(Decimal(value)))
+
+    def test_roman_number_conversion(self):
+        test_data = {
+            3: "III",
+            4: "IV",
+            9: "IX",
+            14: "XIV",
+            21: "XXI",
+            49: "XLIX",
+            99: "XCIX",
+            849: "DCCCXLIX",
+            3000: "MMM",
+            3333: "MMMCCCXXXIII",
```

Comment:
Thanks for writing tests, but I think this test could get more values, right?
For example, all numbers <= 10, something like 999, etc.